packages/api/src/storage.ts

```diff
@@ -314,6 +316,13 @@ interface Lockfile {
  * @see schema.json
  */
 interface LockfileAPI {
+  /**
+   * The date that this SDK was installed.
```

Comment:
i continue to be so sad that `json-schema-to-ts` didn't work out lol

Reply:
i'm already to the point of thinking of building out some script like the `src/packageInfo.ts` build process that creates `schema.json` when we build dists

Reply:
YES